src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -695,6 +695,8 @@ namespace NServiceBus
             "ntedException. Will be removed in version 7.0.0.", true)]
         public static void LoadMessageHandlers<T>(this NServiceBus.EndpointConfiguration config, NServiceBus.First<T> order) { }
     }
+    [System.ObsoleteAttribute("Use `LoadMessageHandlersExtensions` instead. Will be removed in version 7.0.0.", true)]
+    public class static LoadMessageHandlersExtentions { }
```

Comment:
Would this even work? For proper obsoletes you probably need to preserve the extension methods and forward them to the new extension methods. //cc @andreasohlund